src/Sylius/Behat/Context/Ui/Shop/AuthorizationContext.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Ui\Shop;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Element\Shop\Account\RegisterElementInterface;
+use Sylius\Behat\Page\Shop\Account\LoginPageInterface;
+use Sylius\Behat\Page\Shop\Account\RegisterPageInterface;
+use Sylius\Behat\Service\SessionManagerInterface;
+
+final class AuthorizationContext implements Context
+{
+    private LoginPageInterface $loginPage;
+
+    private RegisterPageInterface $registerPage;
+
+    private RegisterElementInterface $registerElement;
+
+    private SessionManagerInterface $sessionManager;
+
+    public function __construct(
+        LoginPageInterface $loginPage,
+        RegisterPageInterface $registerPage,
+        RegisterElementInterface $registerElement,
+        SessionManagerInterface $sessionManager
+    ) {
+        $this->loginPage = $loginPage;
+        $this->registerPage = $registerPage;
+        $this->registerElement = $registerElement;
+        $this->sessionManager = $sessionManager;
+    }
+
+    /**
+     * @When I sign in with email :email and password :password
+     */
+    public function iSignInWithEmailAndPassword(string $email, string $password): void
+    {
+        $this->loginPage->open();
+        $this->loginPage->specifyUsername($email);
+        $this->loginPage->specifyPassword($password);
+        $this->loginPage->logIn();
+    }
+
+    /**
+     * @When I sign in again with email :email and password :password in the previous session
+     */
+    public function iSignInAgainWithEmailAndPasswordInThePreviousSession(string $email, string $password): void
+    {
+        $this->loginPage->open();
+        $this->loginPage->specifyUsername($email);
+        $this->loginPage->specifyPassword($password);
+        $this->loginPage->logIn();
```

Comment:
It does not differ from the previous function at all. Shouldn't it? Or this "previous session" part is done somewhere else?